azurelinuxagent/common/datacontract.py

```diff
@@ -45,7 +45,7 @@ def validate_param(name, val, expected_type):
 
 
 def set_properties(name, obj, data):
-    if isinstance(obj, DataContract):  # pylint: disable=R1705
+    if isinstance(obj, DataContract):
```

Comment:
I think leaving the `else` below in this case is actually better for readability, so anyone reading this code doesn't have to track down every branch below to make sure it returns. So I think I'm good with this change.